auth0_flutter_platform_interface/lib/src/web/web_exception.dart

```diff
@@ -0,0 +1,9 @@
+class WebException implements Exception {
```

Comment:
Can we have this extend `Auth0Exception`? All the mobile exceptions extend it.

Please have the error `code` reference the respective SPA JS error name, e.g. the `MfaRequiredError` error would result in a `WebException` with a `ERR_MFA_REQUIRED` code.

The extra information some of these errors contain (e.g. https://github.com/auth0/auth0-spa-js/blob/master/src/errors.ts#L29) should go in the `details` map of `Auth0Exception`.

Reply:
All of this will lead to a smoother and simpler error handling experience for the developers.

Reply:
Let's also add API docs.

Reply:
My bad, it should be `MFA_REQUIRED`.